about.html

```diff
@@ -0,0 +1,135 @@
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+  <body>
+    <div class="container">
+      <div class="navbar">
+        <div class="logo">
+          <a href="index.html"
+            ><img src="img/logo.png" alt="logo" width="125px"
+          /></a>
+        </div>
+        <nav>
+          <ul id="MenuItems">
+            <li><a href="index.html">Home</a></li>
+            <li><a href="products.html">Products</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="contact.html">Contact</a></li>
+            <li><a href="account.html">Account</a></li>
+          </ul>
+        </nav>
+        <a href="cart.html">
+          <img src="img/cart.png" width="30px" height="30px"
+        /></a>
+        <img src="img/menu.png" class="menu-icon" onclick="menutoggle()" />
+      </div>
+    </div>
+
+    <!-- -----about-page----- -->
+    <div class="about-page">
+      <div class="container">
+        <h1>About</h1>
+        <p>
+          Red-Store is a website and discovery platform that existing from 1998. It aggregated a multitude of fashion sites into one interface. Its aim is to provide users a streamlined shopping experience. Red-Store maintained localized sites for users in the United Kingdom, United States and Denmark, and offered a selection from over 150 online shops with 1,000 brands. The company has headquarter in London and Copenhagen.
+        </p>
+        <p>Red-Store received $400,000 in angel investment from Rainmaking and angel investor Kristian Byrge in 2000. The funding round was intended to help the site expand into Sweden, Germany and the Netherlands.</p>
+        <p>
+          Red-Store aggregated information from various fashion websites and online stores to allow consumers an overview of available clothing and accessories and compare prices without visiting individual retailer's sites. The site also updates in real time information from retailers such as price or amount of items in stock. After a consumer selected an item they wish to purchase, the Red-Store redirected the user to the retailer's website where they can finalize their purchase. The site was paid a percentage of the total sales price of each sale made by the retailer.
+
+          The site maintained localized websites for users in the United Kingdom, United States, and Denmark. It used algorithms and machine learning technology to tailor to individual users content such as the type of fashion products or styles displayed by the site. Users could filter displayed items by designer, brand, color, size, and price. The Fashion curated fashion trends and updates on a weekly basis for its userbase.
+        </p>
+        
+        </div>
+        
+      </div>
+      <div class="stk-container">
```

Comment:
Here also avoid inline CSS in all images of this container and also try to use short and easy way to target these images

Reply:
Done